generic/interbench.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+# Based on code by Brandon Philips <brandon@ifup.org>
+#   copyright: 2008 ifup.org
```

Comment:
Well I can see his first commit to be in `2006` fc9892da4993edc4b7badfa93afe18fac578a1f4 and the `ifup.org` is Brandon's custom page. From https://www.linkedin.com/in/brandonphilips I believe the commit was made while he worked for `IBM`, but I can't guarantee that.